src/FeatureSet.php

```diff
@@ -96,6 +97,11 @@ class FeatureSet
     private $timeGenerator;
 
     /**
+     * @var ValidatorInterface
+     */
+    private $validator;
```

Comment:
Ok. I will then have to add `getValidator()`/`setValidator()` as part of the factory interface, correct? That's what I had and then the description of the factory said it was common functionality to create UUIDs, not necessarily interpret/validate them